recipes/r-pythoninr/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = 'PythonInR' %}
+{% set version = '0.1-4' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-{{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: {{ name }}_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/{{ name }}_{{ version }}.tar.gz
+  sha256: ee9d5c33580b9273d80e43ed4d83dae81de2fc35cc92e310feefe5e4c12c394f
+
+build:
+  number: 0
+  skip: true  # [win32]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+requirements:
+  build:
+    - python
+    - r-base
+    - r-r6
+    - r-pack
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
+
+  run:
+    - python
+    - r-base
+    - r-r6
+    - r-pack
+
+test:
+  commands:
+    - $R -e "library('PythonInR')"  # [not win]
+    - "\"%R%\" -e \"library('PythonInR')\""  # [win]
+
+about:
+  home: https://bitbucket.org/Floooo/pythoninr/
+  license: GPL-3
+  summary: Interact with 'Python' <https://www.python.org/> from within 'R'.
+  license_family: GPL3
+  license_file: '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'  # [unix]
+  license_file: '{{ environ["PREFIX"] }}\Rshare\licenses\GPL-3'  # [win]
```

Comment:
The Windows build [failed](https://ci.appveyor.com/project/conda-forge/staged-recipes/build/1.0.21474/job/v35o2bqnq3l3xh37#L594) because there needs to be a backslash between between R and share, e.g. `R\share`. Could you please tell me which operating system you are using and which script you used (run.sh or run.R) to generate this recipe? I'll fix this if it is a problem with a helper script.

Reply:
```
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.3 LTS
Release:	16.04
Codename:	xenial
```

I used `run.sh`

Reply:
Why bother with these different slashes? All paths in meta.yaml should use normal slashes regardless of the OS you are on.

Reply:
> Why bother with these different slashes?

The [helper script](https://github.com/bgruening/conda_r_skeleton_helper) automatically generated this

>  I'll fix this if it is a problem with a helper script.

@jdblischak - did you determine if this is problem?

Reply:
> did you determine if this is problem?

@gwaygenomics I confirmed this was a recently introduced bug in `run.sh`. I am going to fix it. But this doesn't affect what you do next. You don't need to re-run the helper script. Instead you can either manually insert the backslash (`R\share`) or alternatively, as suggested by @mingwandroid, you could manually insert a forward slash and change the other backslashes to forward slashes as well. Your call.

> Why bother with these different slashes? All paths in meta.yaml should use normal slashes regardless of the OS you are on.

@mingwandroid It was done out of lack of Windows knowledge. I can update our helper script to use forward slashes for the Windows path as well.